aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/BaseRequestIdDecorator.kt

```diff
@@ -66,7 +66,12 @@ abstract class BaseRequestIdDecorator : ClientCodegenDecorator {
     ): List<BuilderCustomization> = baseCustomizations + listOf(RequestIdBuilderCustomization())
 
     override fun extras(codegenContext: ClientCodegenContext, rustCrate: RustCrate) {
-        rustCrate.withModule(ClientRustModule.Types) {
+        rustCrate.withModule(
+            when (codegenContext.settings.codegenConfig.enableNewCrateOrganizationScheme) {
```

Comment:
```suggestion
            // TODO(CrateReorganization): Remove when cleaning up `enableNewCrateOrganizationScheme`
            when (codegenContext.settings.codegenConfig.enableNewCrateOrganizationScheme) {
```

Reply:
I don't think we'll need to comment on each of these since deleting `enableNewCrateOrganizationScheme` will reveal all the locations that need to be fixed.